src/fairplay.js

```diff
@@ -118,6 +118,13 @@ export const defaultGetCertificate = (fairplayOptions) => {
         return;
       }
 
+      if (response.statusCode >= 400 && response.statusCode <= 599) {
+        const cause = String.fromCharCode.apply(null, new Uint8Array(responseBody));
+
+        callback({cause});
+        return;
+      }
```

Comment:
fairplay certificate 4xx and 5xx were never counted as errors apparently.

Reply:
should we wrap this in a `checkForError` function and re-use it?

Reply:
or perhaps emeErrorHandler should take the response and trigger an error on error, and return true so that we can return.

Reply:
Yeah, maybe we can push it into `emeErrorHandler`, though, I'd need to figure out whether the custom getLicense/getCertificate methods's callback get to `emeErrorHandler` or how that piece is wired up.

Also, is it possible that treating 4xx/5xx for these as errors now going to cause an issue compared to previously? My guess is _likely_ no, since if we didn't actually get the cert, it means that playback didn't work.

Reply:
pushing it all into emeErrorHandler might make things more complicated because of existing and custom getLicenses, which expect it to be just err, and license. Though, we could use the function arity to disambiguate between the two, where if we only get 2 arguments, it's the existing stuff, and if it gets 3 items, it's the new thing since we can just pass through the `err, res, resBody`. I'll take a look to see whether it's worth it.